README.md

```diff
@@ -100,7 +103,39 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+class Animal
+
+  attr_accessor :name
+
+  def initialize(name)
+    @name = name
+  end
+
+  def greet
+    puts "Hi"
+  end
+
+end
+
+pumba = Animal.new("Pumba")
+
+class Lion < Animal
+  @@pack = 0
```

Comment:
Will give this to you but the ideal thing to do here would be to see `@@pack` to an array that would contain references to each instance generated. So the initialize method would contain something like this...

```rb
class Lion < Animal
  def initialize
    @@pack << self # where self = instance being generated
  end
end
```